apps/storybook/src/Toolbar.stories.tsx

```diff
@@ -84,60 +90,42 @@ const Template: Story<TemplateProps> = (args) => {
             <ToggleGroup.Btn label="Baz" value="baz" />
           </ToggleGroup>
         </Toolbar>
-      </div>
-
-      <pre style={{ padding: '0 1.5rem' }}>
-        State = {JSON.stringify(allStates, null, 2)}
-      </pre>
-    </>
-  );
-};
-
-export const Default = Template.bind({});
 
-export const Responsive = Template.bind({});
-Responsive.args = { narrow: true };
+        <pre style={{ padding: '0 1.5rem' }}>
+          State = {JSON.stringify(allStates, null, 2)}
+        </pre>
+      </>
+    );
+  },
+} satisfies Story;
 
-export const DocumentInteractions = Template.bind({});
-DocumentInteractions.args = {
-  narrow: true,
-  interactions: [
-    { shortcut: 'Wheel', description: 'Turn' },
-    { shortcut: 'Space', description: 'Accelerate' },
-    { shortcut: 'Ctrl+Alt', description: 'Do a backflip' },
+export const Responsive = {
+  ...Default,
+  decorators: [
+    (Story: StoryFn) => (
+      <div
+        style={{ maxWidth: '30rem', marginLeft: 'auto', marginRight: 'auto' }}
+      >
+        <Story />
+      </div>
+    ),
   ],
-};
+} satisfies Story;
 
-export const OverflowChildren = Template.bind({});
-OverflowChildren.args = {
-  overflowChildren: <Btn label="Some button" onClick={() => {}} />,
-};
+export const DocumentInteractions = {
+  ...Default,
+  args: {
+    interactions: [
+      { shortcut: 'Wheel', description: 'Turn' },
+      { shortcut: 'Space', description: 'Accelerate' },
+      { shortcut: 'Ctrl+Alt', description: 'Do a backflip' },
+    ],
+  },
+} satisfies Story;
 
-export default {
-  title: 'Toolbar/Toolbar',
-  parameters: {
-    layout: 'fullscreen',
-    docs: {
-      source: { type: 'code' },
-      page: () => (
-        <>
-          <Title />
-          <p>
-            The source code of the <em>Default</em> story demonstrates a basic
-            use of the <code>Toolbar</code> component and associated toolbar
-            controls. To reveal the code, click on <em>Show code</em> below the
-            preview.
```

Comment:
I ddin't keep these paragraphs. I think the new documentation page is clear enough.